docs/content/preview/secure/enable-authentication/ysql-login-profiles.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Create login profiles
+headerTitle: Create and configure login profiles in YSQL
+linkTitle: Create login profiles
+description: Create and configure login profiles in YugabyteDB
+headcontent: Prevent brute force cracking with login profiles
+menu:
+  preview:
+    identifier: ysql-login-profiles
+    parent: enable-authentication
+    weight: 725
+type: docs
+---
+
+To enhance the security of your database, you can enable login profiles to lock accounts after a specified number of login attempts. This prevents brute force exploits.
+
+When enabled, database administrators with superuser (or in YugabyteDB Managed, `yb_db_admin`) privileges can create login profiles and assign roles to the profiles.
+
+There is no default profile for roles; you must explicitly assign all roles with login privileges to the profile if you want to policy to apply to all users. Users not associated with a profile continue to have unlimited login attempts.
```

Comment:
if you want to policy to apply to all users  >  if you want the policy to apply to all users